README.md

```diff
@@ -1,2 +1,117 @@
-# proton-local
-Docker-based Proton Network for Development 🛠️
+<p align="center">
+	<a href="https://eosio.lacchain.net">
+		<img src="https://raw.githubusercontent.com/lacchain/eosio.lacchain.net/master/static/img/logos/lacchain-eosio-logo.png"
+		width="400px" >
+	</a>
+</p>
+
+<p align="center">
+	<a href="#">
+		<img src="https://img.shields.io/dub/l/vibe-d.svg" alt="MIT">
+	</a>
+</p>
+
+## Description
+This project builds the Proton network in a local environment so that testing can be done before being published on the public network.
+
+### Why to use a local enviroment?
+Having a local environment provides a series of benefits that you cannot in a public network, for example, with Proton locally, transaction costs are avoided since they are carried out in a development environment and not in production, also, they are accessed to system contracts to modify them as appropriate.
+
+Due to the technology that Blockchain contains, any change in the network creates an immutable record of the action and everything that is modified can affect both positively and negatively the users within it, it is for this reason that it is essential to have an environment premises where functionality tests, performance tests, stress tests, among others, can be carried out without the risk of producing a failure that affects users.
```

Comment:
I would simplify this and say 
In a Blockchain network every transaction creates an immutable record